doc/rst/technotes/attributes.rst

```diff
@@ -14,17 +14,28 @@ Attributes are a mechanism to mark symbols with additional information. This
 information can then be used when processing the abstract syntax tree (AST)
 representation of the source code.
 
-Attributes in Chapel are currently used to indicate feature stability and to
-exclude features from documentation when using :ref:`chpldoc <readme-chpldoc>`.
-The syntax of attributes in Chapel is inspired by Rust's, and includes the
-ability to specify a tool name for an attribute.
-
 Generally, an attribute in Chapel is specified above a symbol declaration with
 an ``@`` followed by an identifier that specifies the attribute name. A list of
 arguments can be enclosed in parentheses ``()``. Arguments can be named or
 specified in order, but mixing the use of named and unnamed arguments in the
 same attribute is not supported.
 
+Attributes in Chapel are currently used to indicate feature stability and to
+exclude features from documentation when using :ref:`chpldoc <readme-chpldoc>`.
+The syntax of attributes in Chapel is inspired by Rust's, and includes the
+ability to specify a tool name for an attribute.
+
+.. code-block:: text
```

Comment:
If it were me, I'd include a little sentence before the code block that says "Here's what an example might look like".